app/models/manager_refresh/inventory_collection_default/physical_infra_manager.rb

```diff
@@ -22,5 +22,51 @@ def physical_server_details(extra_attributes = {})
 
       attributes.merge!(extra_attributes)
     end
+
+    def computer_systems(extra_attributes = {})
+      attributes = {
+        :model_class                  => ::ComputerSystem,
+        :association                  => :computer_systems,
+        :manager_ref                  => [:managed_entity],
+        :parent_inventory_collections => [:physical_servers],
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def hardwares(extra_attributes = {})
+      attributes = {
+        :model_class                  => ::Hardware,
+        :association                  => :hardwares,
+        :manager_ref                  => [:computer_system],
+        :parent_inventory_collections => [:computer_systems],
```

Comment:
the parent should be the top level entity, so  ```:parent_inventory_collections => [:physical_servers]```

(saying that, it should be probably named ```root_inventory_collections```)